src/codeFly/dispatcher/README.md

```diff
@@ -0,0 +1,106 @@
+# Interfaces  
+Here are basic interfaces for front-end team and back-end team to follow:
+
+## Register

```

Comment:
Do we need something like `GET /register HTTP/1.1` that fetches the registration website in the first place?